### PR TITLE
Pac3 sites

### PR DESCRIPTION
**Pull request by erikarasnick**

### CODE REVIEW

File: .github/workflows/build-deploy.yaml [hunk 1]
```diff
@@ -0,0 +1,100 @@
...
+              printf "Dockerfile ${{ matrix.dockerfile[0] }} does not exist"
+              exit 1;
+          fi
+          docker build -f ${{ matrix.dockerfile[0] }} -t ${{ env.container }} .
+          docker images
+
+      - name: Test ${{ matrix.dockerfile[1] }}
+        run: |
+          printf "Testing ${{ env.container }}"
+          docker run --rm -v "${PWD}/test":/tmp ${{ env.container }} my_address_file_geocoded.csv
```
Comment (cole-brokamp) [inline .github/workflows/build-deploy.yaml:72]: We need to add an example site argument for the test here, right?

File: README.md [hunk 2]
```diff
@@ -52,27 +52,53 @@ Lurie Children's Center in Lincoln Park | `lcclp`
...
+UNC Hospitals Children's Specialty Clinic	| `unc`
+Barbara Bush Children's Hospital at Maine Medical	| `maine`
 
 Example calls (that will work with example file included in repository):
 
 **MacOS**
 
 ```
-docker run --rm -v "$PWD":/tmp degauss/pepr_drivetime:0.6 my_address_file_geocoded.csv cchmc
+docker run --rm -v "$PWD":/tmp degauss/drivetime:0.7 my_address_file_geocoded.csv cchmc
```
Comment (cole-brokamp) [inline README.md:87]: let's make this release be 1.0

**Review by cole-brokamp — CHANGES_REQUESTED** [review]
after you fix test in GHA and make it version 1.0, I think we are good to go!

**Comment by cole-brokamp** [comment]
what's up with the permission denied error in the gha build?

can we also resolve the startup messages for some packages that they were built under a different version of R?  I think we should resolve using renv, right?

**Comment by erikarasnick** [comment]
This is the same issue I was having before. I have to manually give the original repo write permissions for the package. I updated and re-ran actions but I think there is a lag, so re-running again. 

Oops, I noticed those startup messages and meant to get rid of them. I'm not sure why they say built under R 4.0.5 since I have R 4.0.4 which matches the Dockerfile and renv.lock ...

**Comment by cole-brokamp** [comment]
that worked for suppressing the warning messages on package load.  whenever you figure out the authentication issue, let's release

**Comment by erikarasnick** [comment]
build-deploy.yaml was still pointing to the pm packages repo, so that explains the permissions issue.

**Pull request merged**